app/scenes/right_menu_drawer/right_menu_drawer_item.js

```diff
@@ -23,18 +23,23 @@ export default class MainMenuItem extends React.Component {
         style: React.PropTypes.oneOfType([
             React.PropTypes.object,
             React.PropTypes.array
-        ])
+        ]),
+        shouldRender: React.PropTypes.bool
```

Comment:
Can you rename this prop to `show`? That'll make it consistent with a lot of code in the webapp